rules/falco_rules.yaml

```diff
@@ -3083,6 +3083,41 @@
   priority: CRITICAL
   tags: [syscall, mitre_defense_evasion]
 
+- list: ingress_remote_file_copy_binaries
+  items: [wget]
+
+- macro: ingress_remote_file_copy_procs
+  condition: (proc.name in (ingress_remote_file_copy_binaries))
+
+# Users should overwrite this macro to specify conditions under which a
+# Custom condition for use of ingress remote file copy tool in container
+- macro: user_known_ingress_remote_file_copy_activities
+  condition: (never_true)
+
+-  macro: curl_download
+   condition: proc.name = curl and
+              (proc.cmdline contains " > " or
+              proc.cmdline contains " >> " or
+              proc.cmdline contains " | " or
+              proc.cmdline contains " -o " or
+              proc.cmdline contains " --output " or
+              proc.cmdline contains " -O " or
+              proc.cmdline contains " --remote-name ")
+
+- rule: Launch Ingress Remote File Copy Tools in Container
+  desc: Detect ingress remote file copy tools launched in container
+  condition: >
+    spawned_process and
+    container and
```

Comment:
neat: I would suggest the `user_known_ingress_remote_file_copy_activities` become the overall control to the rule. That is:
```
spawned_process and
container and 
(ingress_remote_file_copy_procs or curl_download) and
not user_known_ingress_remote_file_copy_activities
```
Let me know if the above changes are better.

Really appreciate your patience. You did a really good here!

Reply:
No worries. The feedback has been spot on and I've learned a lot.